.github/workflows/publish-release-container.yml

```diff
@@ -9,40 +9,29 @@ jobs:
   build:
     name:                         Build and push image
     runs-on:                      ubuntu-latest
-    strategy:
-      matrix:
-        # this matrix runs the jobs twice, in parallel. This is due to
-        # both buildah actions can't manage more than one image tag.
-        tag:
-          - latest
-          - ${{ github.event.release.tag_name }}
-    env:
-      REGISTRY:                   docker.io/parity
-      IMAGE_NAME:                 substrate-api-sidecar
-      VERSION:                    ${{ github.event.release.tag_name }}
-
     steps:
+
     - uses:                       actions/checkout@v2
 
-    - name:                       Build Image tagged ${{ matrix.tag }}
-      uses:                       redhat-actions/buildah-build@v1
+    - name:                       Build Image
+      uses:                       redhat-actions/buildah-build@v2.2
       with:
-        image:                    ${{ env.IMAGE_NAME }}
-        tag:                      ${{ matrix.tag }}
+        image:                    substrate-api-sidecar
+        tags:                     ${{ github.event.release.tag_name }}, latest
```

Comment:
Just to understand better, is this building two images simultaneously or just one image with two tags?

Reply:
The previous version did build two images simultaneously, this now just adds tags.